content/metadata.yaml

```diff
@@ -1397,12 +1397,8 @@ authors:
   - 
     github: aimundo
     name: Ariel I. Mundo
-        affiliations:
-          - institution: University of Arkansas
-            department: Department of Biomedical Engineering
-            city: Fayetteville
-            state: AR
-            country: USA
+    affiliations:
+    - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
```

Comment:
The spacing is different here, but the AppVeyor artifact looks okay so it didn't matter.